packages/commands/package.json

```diff
@@ -0,0 +1,45 @@
+{
+	"name": "@wordpress/commands",
+	"version": "0.1.0",
+	"description": "Handles the commands menu.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"gutenberg",
+		"commands",
+		"k"
```

Comment:
Hey @youknowriad :wave: What does this "k" keyword mean? Noticed this when reviewing #52987, when looking up what are the dependencies of the commands packages 🙂

Reply:
most packages call these cmd+k menus I guess :)

Reply:
I see, so it's not a typo 🙂 I haven't found any other packages using this keyword though.

Reply:
Fine to remove.